### DISCUSSION TITLE
Face - Recognition

### DISCUSSION THREAD

**Discussion by ruhaanpathan** [community/community#166683] [Other Feature Feedback, Questions, & Ideas]
Project Definition: Create Image Recognition App in Python

Description:Image recognition is a technology that will make our lives easier. The project aims to create an image recognition app using Python. This app will use computer vision and image processing algorithms to recognise objects in images and then provide the user with the object’s name. This technology has many applications, such as recognising faces and identifying people in images or videos, recognising scenes and objects in images or videos, identifying text in images or videos, etc.

**Comment by Ashutosh-177** [comment]
You have a brilliant project in mind.
This is a part of supervised learning in machine learning. You have to go through the CNN (convolutional neural network). Use/import open CV.
Python version of 3.7+ must be used for face recognition. 
TensorFlow can be used and here are the layers:
1. Conv2D: Convolution layer to extract features.
2. MaxPooling2D: Reduces spatial dimensions (downsampling)
3. Flatten: Converts 2D features to 1D before dense layers
4. Dense: Fully connected layers
5. Dropout: it discard neurons which prevents overfitting in model

Go through this topics ... It will help you in project completion 
For further queries here is my mail ID: ashutoshhjp1067@gmail.com

**Reply by ruhaanpathan** [reply]
Thanks for insightfull view
